test/automation-bot.ts

```diff
@@ -640,7 +640,7 @@ describe('AutomationBot', async () => {
             triggerId = event.args.triggerId.toNumber()
         })
 
-        describe('command update', async () => {
+        describe.skip('command update', async () => {
```

Comment:
I'd remove the test since update of the command will not be possible.
Unless we go for a different solution with command versioning